src/mixed.js

```diff
@@ -538,7 +548,12 @@ const proto = (SchemaType.prototype = {
         .filter(
           (n, idx, list) => list.findIndex(c => c.name === n.name) === idx,
         ),
-    };
+    }
+
+    if (next._whitelist.size) description.whitelist = next._whitelist.describe();
```

Comment:
lets call these `oneOf` and `notOneOf` to match the method name, it'll involve a little less mental overhead mapping schema to descriptions etc

Reply:
Sure thing @jquense